MetaMask/metamask-extension
# fix: update controller utils package to v5.0.2 and all related packages

## **Description**
- Updated `@metamask/controller-utils` package to v5.0.2 to update the linea goerli explorer url
- Updated all npm packages that use this `@metamask/controller-utils` package

## **Manual testing steps**

1. Step1: Send a transaction using the Linea Goerli network
2. Step2: Open the transaction description
3. Step3: Click on the "View on Etherscan" button
4. Step4: Check that it redirects to https://goerli.lineascan.build

## **Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change._

### **Before**

_[screenshot]_

### **After**

_[screenshot]_

## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
